Bugzilla/Search.pm

```diff
@@ -2318,10 +2319,10 @@ sub _date_translate {
 sub SqlifyDate {
   my ($str) = @_;
   my $fmt = "%Y-%m-%d %H:%M:%S";
+  my $date = DateTime->now();
```

Comment:
Should we be using the database's current time rather than the webheads?

Reply:
All pods should be in sync time wise but there could be some difference. I thought about it and was trying to keep the extra SQL calls to a minimum to get the current timestamp each time SqlifyDate is called. But maybe in would be insignificant? Thoughts?

Reply:
I take that back, the MySQL server itself is a separate instance and may not be time synced.

Reply:
Fair enough, let's leave it client-side.